third-party/frugally-deep/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 therock_subproject_fetch(therock-frugally-deep-sources
   CMAKE_PROJECT
-  # Originally mirrored from: https://github.com/Dobiasd/frugally-deep/archive/refs/tags/v0.16.2.tar.gz
-  URL https://rocm-third-party-deps.s3.us-east-2.amazonaws.com/frugally-deep-0.16.2.tar.gz
-  URL_HASH SHA256=b16af09606dcf02359de53b7c47323baaeda9a174e1c87e126c3127c55571971
+  # TODO: copy tarball to rocm-third-party-deps.s3.us-east-2.amazonaws.com
```

Comment:
Happy to upload the specified version but can you elaborate why the downgraded is needed?

Reply:
@marbre , I've amended the PR with a comment with links to the upstream MIOpen issue.

Reply:
@marbre as @scottt mentioned, more context available at https://github.com/ROCm/MIOpen/issues/3588#issuecomment-2715008600

Reply:
Thanks for pointing me to it. Let me upload the artifact for you asap.

Reply:
@stellaraccident , What are the other consumers of `frugally-deep` in the ecosystem? I was optimistic that MIOpen was the only one.

Reply:
I think in ROCm that is the only one. But it is not vendored consistently so  think we've not seen the last of this issue.

Reply:
Agreed. @BradPepersAMD if you're not aware of this yet, you probably should be.

Reply:
```suggestion
```
Uploaded it, thus you can drop the TODO :)

Reply:
@marbre , commit added.

Reply:
:+1: seems you still need to drop the TODO note.

Reply:
Ah, @marbre  thanks for catching that. PR updated.